index.rst

```diff
@@ -100,73 +110,268 @@ Data Sets Types and Goals
+
+2. testdata_ci_hsc
+
+   The `testdata_ci_hsc` package (https://github.com/lsst/testdata_ci_hsc) includes just enough data to exercise the main steps of the current pipeline: single-frame processing, coaddition, and coadd processing.  The input data comprises 33 raw images from 12 HSC visits in r and i band, pre-made master darks, dome flats, sky flats, biases and detector defect files for these, and the necessary subset of the PS1-PV3 reference catalog.  These data total 8.3 GB.  The `ci_hsc` package is run to process the `testdata_ci_hsc` data automatically on a nightly basis by the CI system and can be explicitly included in developer-initiated CI runs on development branches.  The package also includes some simple tests to make sure that the expected outputs exist, but practically no tests of algorithmic or scientific correctness.  Both by name and content, this is a CI-level dataset as defined above.
+
+SMALL
+=====
+1. https://github.com/lsst/validation_data_hsc
+    - 56 GB raw + master calibrations.
+    - The entire `validation_data_hsc` repo is 250 GB because it includes a set of processCcd+coadd processed data.
+    - Calibration data available as pre-computed masters and used to do ISR.
+    - Currently processed on a daily (8 hour?) cadence through to coadd.
+    - Currently not used for DIA.
+
+MEDIUM
+======
+1. HSC RC2
+
+   The "RC2" dataset consists of two complete HSC SSP-Wide tracts and a single HSC SSP-UltraDeep tract (in the COSMOS field).  This dataset is  processed every two weeks using the weekly releases of the DM stack.  The processing includes the entire current DM pipeline (including `jointcal`, which is not included in `ci_hsc`) as well as the `pipe_analysis` scripts, which generate a large suite of validation plots, and an uplodate of the results of `validate_drp` to SQuaSH.  Processing currently requires some manual supervision, but we expect processing of this scale to eventually be fully automated.  See also https://confluence.lsstcorp.org/display/DM/Reprocessing+of+the+HSC+RC2+dataset
+
+   The HSC RC2 data is presently (2019-09-10) available at NCSA at in `/datasets/hsc/repo`.  The HSC dataset was defined in a JIRA ticket: `Redefine HSC "RC" dataset for bi-weeklies processing <https://jira.lsstcorp.org/browse/DM-11345>`_
+
+   Lauren MacArthur spent significant time into the defining this dataset to consist of both mostly good data plus some specific known more challenging cases.  Explicitly increasing the proportion of more challenging cases increases the efficiency of identifying problems for a fixed amount of compute resources at the expense of making the total scientific performance numbers less representative of a full set of data.  This is a good tradeoff to make, but also an important point to keep in mind when using the processing results of such datasets to make predictions of performance of the LSST Science Pipelines on LSST data.
```

Comment:
"In defining this dataset, Lauren MacArthur paid particular attention to having it consist of..." 
or leave out specific reference to me
"Particular attention was paid in defining this dataset for it to consist of..."

"...more challenging cases (see above JIRA issue for details)."

"...less representative of **the average quality for a full --survey-sized--** set of data."